es6-talk.md

```diff
@@ -1,3 +1,28 @@
-lalala
+### Purpose
 
-dump ideas here.
+ES6/ES2015 is the present. Its already in browser implementation and the TC39 team is moving at good pace to add more features to the JavaScript language.
+Relatively low learning curve and babel/polyfills guarantee no extra effort to support older browsers.The newer features being added are building upon the features present in ES2015.
+Its a good idea to be using it in your projects now and try to include it in your old projects when refactoring or adding new features.
+
+### Outline
+
+**Introduction**
+Explain common avenues where ES2015 can be easily used and how it will affect ease of programming and induce lesser confusions and mistakes.
```

Comment:
Hey wow! This would be lot of fun. Did you perform this live? How was the response?